cmd/init_mon.go

```diff
@@ -40,7 +40,7 @@ const (
 [global]
 fsid = %s
 mon initial members = %s
-mon host = v2:127.0.0.1:3300/0
+mon host = 127.0.0.1
```

Comment:
Actually, the syntax is: `[v2:127.0.0.1:3300,v1:127.0.0.1:6789]`. Let's try it.

Reply:
Updated